pkg/yttlibrary/overlay/match_annotation_expects_kwarg.go

```diff
@@ -73,33 +97,33 @@ func (a MatchAnnotationExpectsKwarg) checkValue(val interface{}, matches []*file
 		}
 		if typedResult, ok := result.(starlark.Bool); ok {
 			if !bool(typedResult) {
-				return fmt.Errorf("Expectation of number of matched nodes failed")
+				return MatchAnnotationNumMatchError{"Expectation of number of matched nodes failed"}
```

Comment:
i think this should be plain error, not MatchAnnotationNumMatchError since called function returned gibberish.

Reply:
oops. never mind about this.